### PR TITLE
adding font-variation-settings

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

File: css/properties.json [hunk 1]
```diff
@@ -3240,6 +3240,26 @@
     ],
     "status": "standard"
   },
+  "font-variation-settings": {
+    "syntax": "normal | [ &lt;string&gt; &lt;number&gt; ] #",
```
Comment (SebastianZ) [inline css/properties.json]: Nit: We're not adding spaces between `]` and the multiplier. Can you please remove it?

Sebastian

File: css/properties.json [hunk 2]
```diff
@@ -3240,6 +3240,26 @@
...
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
```
Comment (SebastianZ) [inline css/properties.json]: The [spec. says](https://drafts.csswg.org/css-fonts-4/#low-level-font-variation-settings-control-the-font-variation-settings-property):

> Animating font-variation-settings is possible using a mechanism similar to animating the transform property: Two declarations of font-feature-settings can be animated between if they are "like".

So, `discrete` is wrong here. There needs to be a short description added to l10n/css.json explaining the animation type and a referencing keyword here.

Sebastian

File: css/properties.json [hunk 3]
```diff
@@ -3240,6 +3240,26 @@
...
+    "percentages": "no",
+    "groups": [
+      "CSS Fonts"
+    ],
+    "initial": "<code>normal</code>",
+    "appliesto": "allElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
```
Comment (SebastianZ) [inline css/properties.json]: Please use `perGrammar` here. (It's actually the same, though it's what the spec. says.)

Sebastian

File: css/properties.json [hunk 4]
```diff
@@ -3240,6 +3240,26 @@
...
+    "alsoAppliesTo": [
+      "::first-letter",
+      "::first-line",
+      "::placeholder"
+    ],
+    "status": "standard"
```
Comment (SebastianZ) [inline css/properties.json]: As the spec. is still an Editor's Draft, I think this should be marked as `experimental`, even when there are already multiple implementations.

Sebastian

**Comment by chrisdavidmills** [comment]
I've attempted to update the data as per your comments. Let me know if this is ok. Cheers for the review!

**Pull request merged**